examples/pytorch/AxHyperOptimizationPTL/requirements.txt

```diff
@@ -0,0 +1,7 @@
+torch>=1.6.0
```

Comment:
Do we need `requirements.txt` ? the packages are already part of `conda.yaml`

Reply:
Requirements.txt is only needed for TorchServe deployment part. In the examples for Ax/Botorh we don't need these as it only covers the model training part